acceptance/tests/fixtures/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM kschoche/http-echo:latest
```

Comment:
This adds curl to the http-echo image, I plan to swap out this for the actual http-echo image, so this can be ignored for now in review-- the only change would be to delete this file and swap the image.

Reply:
Swapped and deleted